src/Pages/junghyun/Main/Main.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import Comment from '../Main/components/Comment';
+import './Main.scss';
+
+class Mainjunghyun extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      inputComment: '',
+      commentList: [],
+    };
+  }
+
+  handleCommentInput = e => {
+    if (e.key === 'Enter') {
+      this.addComment();
+    } else {
+      this.setState({
+        inputComment: e.target.value,
+      });
+    }
+  };
+
+  addComment = () => {
+    this.setState({
+      commentList: this.state.commentList.concat(this.state.inputComment),
+      inputComment: '',
+    });
+  };
+
+  render() {
+    return (
+      <>
+        <header className="Headerjunghyun">
+          <div className="westagramImageAndLogo">
+            <div className="westagramImage">
+              <i className="fab fa-instagram"></i>
+            </div>
+            <div className="westagramLogo">
+              <a>
+                <h1>Westagram</h1>
+              </a>
+            </div>
+          </div>
+          <div className="inputAndSearch">
+            <i class="fas fa-search"></i>
+            <input type="text" placeholder="검색" />
+          </div>
+          <nav className="topRightMenu">
+            <a href="#">
+              <i class="fas fa-compass"></i>
+            </a>
+            <a href="#">
+              <i class="far fa-heart"></i>
+            </a>
+            <a href="#">
+              <i class="fas fa-user"></i>
+            </a>
+          </nav>
+        </header>
+        <main className="Mainjunghyun">
+          <div className="feedOutline">
+            <article className="innerFeedOutline">
+              <section className="feedHead">
+                <div className="userImageAndName">
+                  <i class="far fa-user-circle"></i>
+                  <p>canon_mj</p>
+                </div>
+                <i class="fas fa-ellipsis-h"></i>
+              </section>
+              <div>
+                <img
+                  className="feedImage"
+                  alt="scone bread image"
+                  src="/images/junghyun/img_baking.jpeg"
+                />
+              </div>
+              <section>
+                <ul className="comments">
+                  <li className="commentFirst">
+                    canon_mj 위워크에서 진행한 베이킹 클래스...
+                  </li>
+                  <li className="commentSecond">
+                    neceosecius 거봐 좋았잖아~~~~
+                  </li>
+                </ul>
+                <div className="commentList">
+                  {this.state.commentList.map(el => {
+                    return <Comment newComment={el} />;
+                  })}
```

Comment:
```suggestion
                  {this.state.commentList.map(el => <Comment newComment={el} />)}
```
arrow function에서 return이 바로 실행될 경우는 중괄호와 return keyword 생략 가능합니다!

Reply:
그리고 위에 두개 기본으로 있는 댓글들같은 경우에도 하드코딩으로 넣어놓지 마시고 state값에 따라서 동일하게 생성되도록하는게 좋아보입니다!

state의 초기값에 댓글내용 2개 추가해놓으면 하드코딩안해놔도 자연스럽게 생성될것으로 보입니다!